nodejs/server.js

```diff
@@ -11,10 +10,11 @@ const srv = http.createServer( (req, res) => {
     status = 404;
   }
   res.writeHead(status, {
-    'Content-Type': 'text/plain; encoding=utf-8',
-    'Content-Length': data.length});
+      'Content-Type': 'text/plain; encoding=utf-8',
```

Comment:
Thanks you. Copy-pasted from: https://github.com/squeaky-pl/japronto/blob/7fdb8218bf9486da8490c02c0171cfe98fea5c44/benchmarks/nodejs/micro.js